keyboards/ergodox/keymaps/narze/config.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CONFIG_H_
+#define CONFIG_H_
+
+#include "../../config.h"
+
+#undef TAPPING_TERM
+#define TAPPING_TERM 150
+
+// Combos not working yet
+// #define COMBO_TERM 20
+// #define COMBO_COUNT 1
+// #define COMBO_ALLOW_ACTION_KEYS
+
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This is not needed anymore, as it's no longer used.  It is now the default behavior. 

```suggestion
// #define STRICT_LAYER_RELEASE
```